code/game/objects/structures/plasma_geyser.dm

```diff
@@ -0,0 +1,42 @@
+
+/obj/structures/plasmageyser
+	name = "Plasma Geyser"
+	desc = "A mound of basalt rock, erupting with bubbling molten plasma. It constantly emits toxic fumes."
+	anchored = TRUE
+	icon = 'icons/obj/lavaland/geyser.dmi'
+	icon_state = "geyser_plasma"
+	resistance_flags = LAVA_PROOF | FIRE_PROOF
+	armor = list(MELEE = 30, BULLET = 80, LASER = 90, ENERGY = 90, BOMB = 80, RAD = 100, FIRE = 100, ACID = 100)
+
+/obj/structures/plasmageyser/Initialize()
+	START_PROCESSING(SSprocessing, src)
+	. = ..()
+
+// for sanity checks
+/obj/structures/plasmageyser/Destroy()
+	STOP_PROCESSING(SSprocessing, src)
+	. = ..()
```

Comment:
```suggestion
/obj/structures/plasmageyser/Initialize()
	. = ..()
	START_PROCESSING(SSprocessing, src)

// for sanity checks
/obj/structures/plasmageyser/Destroy()
	STOP_PROCESSING(SSprocessing, src)
	return ..()
```

First is a functional change, second is just styling to signify **nothing** should go below that call.